jwst/pipeline/calwebb_coron3.py

```diff
@@ -118,9 +124,7 @@ def process(self, input):
                 suffix='psfsub', acid=acid
             )
 
-            # Create a ModelContainer of the psf_sub results to send to
-            # outlier_detection
-            self.log.debug('Building ModelContainer of klip results')
+            # Create a ModelContainer of the psf_sub results.
```

Comment:
So, for my understanding, it's still necessary to build a `ModelContainer` in order to send the proper "stack" of inputs to the `resample` step? It looks like the `ModelContainer` contains a stack of psf-subtracted images from all integrations from all target exposures, right? Rather than sending to `resample` a `ModelContainer` consisting of multiple `CubeModels`, each of which has it's own stack of images (per integration).

Reply:
I'll dive into this a bit more and see if a simplification can happen; less transformation the better.

Reply:
To resample everything down to a single image, the integrations need to be unwrapped and placed in a single-level list. However, I have refactored this bit to be more generic.